.github/workflows/ci.yaml

```diff
@@ -40,11 +40,15 @@ jobs:
       - name: Install ffigen
         run: dart pub global activate ffigen ^5.0.0 # TODO migrate to 6.x later
 
-      - name: Install dart dependencies
+      - name: Install dart dependencies (single block)
         working-directory: ./frb_example/pure_dart/dart
         run: dart pub get
 
-      - name: Run valgrind script
+      - name: Install dart dependencies (multi blocks)
```

Comment:
> So, in `valgrind_test`, the multi-blocks example has its dependency installed, but never executed? :(
> 
> In other words, we should have somewhere executing the multi-file package's tests.

Before, you said there is no need to do valgrind_test for the multi version. So there is only ` - name: Run valgrind script (single block)`. And the corresponding `run.sh` and `valgrind_util.py` are removed from `./frb_example/pure_dart_multi`.

Reply:
Sure, no need to *valgrind* test the multi version; but need a simple `dart test` for the pure_dart_multi. Otherwise how can we ensure the code is right :)

Reply:
> Sure, no need to _valgrind_ test the multi version; but need a simple `dart test` for the pure_dart_multi. Otherwise how can we ensure the code is right :)

I've added a simple test in run.sh within 0ef57e52004841b204416f54f100f42624f18955.